### ISSUE TITLE
[RFC] Add the request information in the exception pages

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony#26186]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | yes
| Symfony version  | 4.1

**Would you find it useful to add the request information in the exception pages?** The idea would be to add a `Request` tab here:

![symfony-exception](https://user-images.githubusercontent.com/73419/36249961-b05c0600-123c-11e8-9019-795d665e122f.png)

The information would be the same as displayed by Sentry ...

... the raw request headers:

![request-details](https://user-images.githubusercontent.com/73419/36250027-e6771f4a-123c-11e8-9923-64f71a5cf0be.png)

... and the cURL formatted request to allow you to reproduce it:

![request-curl](https://user-images.githubusercontent.com/73419/36250030-ebef5938-123c-11e8-9273-3067996ef799.png)

-----

I don't know if it's possible to do this and how complex it would be. I'm just asking to check the interest from the community. Thanks!

**Comment by iltar** [comment]
In theory the information can already be found in the request panel, however, it would make it easier to _also_ display it here.

**Comment by sroze** [comment]
Yes, definitely agree with you @javiereguiluz!

**Comment by Simperfit** [comment]
+1

**Comment by phansys** [comment]
I agree this can be really useful. Even, if the CURL request can be displayed, a "Retry" button could be a nice to have addition. But IMO, this feature should be also present in the "Request" section regardless a exception occurred or not.

:+1:

**Comment by Simperfit** [comment]
@javiereguiluz do you want (and have time) to implement it ?
